src/redux/actions/experiments/createCellLevelMetadata.js

```diff
@@ -0,0 +1,31 @@
+import fetchAPI from 'utils/http/fetchAPI';
+import handleError from 'utils/http/handleError';
+import dayjs from 'dayjs';
+import { loadSamples } from '../samples';
+import loadExperiments from './loadExperiments';
+
+const createCellLevelMetadata = (experimentId, body) => async (dispatch) => {
+  const createdAt = dayjs().toISOString();
```

Comment:
the createdAt part should be automatically set in sql, so no need to generate it or send it in the endpoint